core/trino-spi/src/test/java/io/trino/spi/predicate/BenchmarkSortedRangeSet.java

```diff
@@ -169,6 +169,36 @@ private long benchmarkContainsValue(List<SortedRangeSet> dataRanges)
         return result;
     }
 
+    @Benchmark
+    public List<Integer> getOrderedRangesSmall(Data data)
+    {
+        return benchmarkGetOrderedRanges(data.smallRanges);
+    }
+
+    @Benchmark
+    public List<Integer> getOrderedRangesLarge(Data data)
+    {
+        return benchmarkGetOrderedRanges(data.largeRanges);
+    }
+
+    private List<Integer> benchmarkGetOrderedRanges(List<SortedRangeSet> dataRanges)
+    {
+        List<Integer> result = new ArrayList<>(dataRanges.size());
+        for (int index = 0; index < dataRanges.size(); index++) {
+            int hash = 0;
```

Comment:
why we are computing hash code instead of just collecting `Objects`?

Reply:
So that we don't create very long result list, while still iterating over returned data and using it

Reply:
Why not to just use `+`?

Reply:
does it matter?

Reply:
it's a waste, isn't it? Probably it doesn't affect results, but still